config/routes.rb

```diff
@@ -6,6 +6,8 @@
       sessions: "jobseekers/sessions",
     }
 
+    get "/jobseekers/saved_jobs", to: "jobseekers/saved_jobs#index", as: :jobseeker_root
```

Comment:
Can't go in `namespace :jobseekers` because that would prepend 'jobseekers' to the path helper method name.